schema/src/lib.rs

```diff
@@ -194,7 +195,17 @@ macro_rules! impl_schema_int {
     )*};
 }
 
-impl_schema_int!(u128, u64, u32, u16, u8, i128, i64, i32, i16, i8,);
+impl_schema_int!(u128, u64, u32, u16, u8, i128, i64, i32, i16, i8, usize,);
```

Comment:
hmm, as which type is usize serialized? and is it the same type for both parity and serde?

Reply:
I should add that `usize` is not quite portable

Reply:
That's not supposed to be in this PR.

Reply:
I'm not asking you to add anything, I'm asking you to remove `usize` because I don't see how it can be serialized dependably on all platforms

Reply:
Marin, I don't understand what you mean here. The `IntoSchema` for `usize` leaked through, from an old stash, it's

> not supposed to be in this PR.

As in it got included by mistake. Thank you for catching it.